src/test/java/org/testng/internal/ParametersTest.java

```diff
@@ -15,6 +15,7 @@
 
   @Test
   @Parameters({"testdata"})
+  @SuppressWarnings("unused")
```

Comment:
I think the warning is already removed once you moved the file in the test folder

Reply:
No it's not. Its basically for two parameters of this method viz., `xmlTest` and `testdata` which are declared but not used in the test method.